src/cloud/internal/cloudservice.cpp

```diff
@@ -78,9 +79,12 @@ void CloudService::init()
     m_oauth2->setAuthorizationUrl(configuration()->authorizationUrl());
     m_oauth2->setAccessTokenUrl(configuration()->accessTokenUrl());
     m_oauth2->setModifyParametersFunction([](QAbstractOAuth::Stage stage, QVariantMap* parameters) {
-        if (stage == QAbstractOAuth::Stage::RequestingAuthorization) {
```

Comment:
why droppiung that `if`?

Reply:
to have send information about app version in other requests(token, refresh) besides authorization

Reply:
OK

Reply:
Still need to fix the code style issues...